index.bs

```diff
@@ -438,4 +433,4 @@ A file containing a 'pict' track compliant with this profile is expected to list
 <h2 id="change-list">Changes since v1.1.0 release</h2>
     - EDITORIAL: <a href="https://github.com/AOMediaCodec/av1-avif/pull/223">Stop using `dfn value` for definitions.</a>
     - EDITORIAL: <a href="https://github.com/AOMediaCodec/av1-avif/pull/222">Add assert-ids in the spec for conformance file testing and ComplianceWarden</a>
-
+    - <a href="https://github.com/AOMediaCodec/av1-avif/pull/228">Remove recommendations regarding still flags in image items.</a>
```

Comment:
Nit: add "picture" after "still"

It would also be good to say "still picture" in the commit message, if it does not exceed the recommended 50-character line length.

Reply:
> Nit: add "picture" after "still"

Done.

> Should the commit message say "Fixes https://github.com/AOMediaCodec/av1-avif/issues/216"?

I usually avoid that because it triggers a GitHub cross-reference at every rebase. I can insert it at squashing time if you believe it is worth it. Note that the default squash title contains the PR number.

> It would also be good to say "still picture" in the commit message, if it does not exceed the recommended 50-character line length.

I am planning on squashing as "Replace AV1 still_pic=1 recommendation by note".